app/config.py

```diff
@@ -412,3 +412,5 @@ def sl_getenv(env_var: str, default_factory: Callable = None):
 
 PHONE_PROVIDER_2_HEADER = os.environ.get("PHONE_PROVIDER_2_HEADER")
 PHONE_PROVIDER_2_SECRET = os.environ.get("PHONE_PROVIDER_2_SECRET")
+
+ZENDESK_HOST=os.environ.get('ZENDESK_HOST', 'noone.zendesk.com')
```

Comment:
Should we also need a ZENDESK secret/token here?

Reply:
No need. Zendesk tickets can be opened by anyone without having a zendesk account.